### PR TITLE
Taskextensions sample notebook

### PR DESCRIPTION
**Pull request by jesshill99**
Removed Cast<T> example (as this extension method is no longer useful)

### CODE REVIEW

File: Corvus.Extensions.Samples/TaskExtensionsSample.dib [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+#!csharp
+
+Task t = Task.FromResult(new Derived());
+
+Task<Base> tb = (Task<Base>)t;
+tb.Result
+
+#!markdown
+
+Using `CastWithConversion<>()` allows us to cast the task to a base.
```
Comment (jongeorge1) [inline Corvus.Extensions.Samples/TaskExtensionsSample.dib:67]: Couple of comments on this line.

Firstly, if "task" and "base" refer to .NET types, capitalize them and format them using the backtick so they appear as `Task` and `Base`.

Secondly, the cell doesn't cast the task to a base. It casts the `Task` to a `Task<Base>`.

File: Corvus.Extensions.Samples/TaskExtensionsSample.dib [hunk 2]
```diff
@@ -0,0 +1,72 @@
+#!markdown
+
+## TaskExtensions - Sample notebook
+
```
Comment (jongeorge1) [inline Corvus.Extensions.Samples/TaskExtensionsSample.dib:4]: Our other sample notebooks have a section titled "How to use .NET Interactive notebooks" at the beginning - we should include that here for consistency.

Comment (jongeorge1) [reply Corvus.Extensions.Samples/TaskExtensionsSample.dib:4]: Never mind, I've just found the pull request that removes all of these sections and puts them in a central README.md 🙂

**Review by jongeorge1 — APPROVED** [review]

File: Corvus.Extensions.Samples/TaskExtensionsSample.dib (hunk 1 above)
Comment (jesshill99) [reply Corvus.Extensions.Samples/TaskExtensionsSample.dib]: Fixed ✅

**Review by idg10 — APPROVED** [review]

**Pull request merged**